Bug in UI on homepage #107

Jwright707 opened the pull request:
It appears that the issue comes from the Button.module.scss file. The import path wasn't correct which could lead to some of the styles to not come through. I debugged this using your main site because I was having trouble reproducing this on my local machine. The way I reproduced this on prod was to go from Home -> Freshers Section -> Mentors -> Home. And each page I scrolled up and down too.

apanjain commented:
You can replicate this on your local machine by creating a production build
```sh
cd frontend/
yarn build
npx serve build/
```

Absolute imports are not an issue in React. [See here](https://create-react-app.dev/docs/importing-a-component/#absolute-imports)
Try looking into the display properties instead

Jwright707 commented:
Hi @apanjain, thank you for the feedback and advice. I've reverted that import and added the correct fix. Thank you again and hope to hear from you soon 😀

apanjain commented:
@Jwright707 Issue is resolved now. Thanks!
